src/lines/LineSegmentsGeometry.js

```diff
@@ -83,10 +83,10 @@ class LineSegmentsGeometry extends InstancedBufferGeometry {
       colors = new Float32Array(array)
     }
 
-    const instanceColorBuffer = new InstancedInterleavedBuffer(colors, 6, 1) // rgb, rgb
+    const instanceColorBuffer = new InstancedInterleavedBuffer(colors, itemSize * 2, 1) // rgb(a), rgb(a)
 
-    this.setAttribute('instanceColorStart', new InterleavedBufferAttribute(instanceColorBuffer, 3, 0)) // rgb
-    this.setAttribute('instanceColorEnd', new InterleavedBufferAttribute(instanceColorBuffer, 3, 3)) // rgb
+    this.setAttribute('instanceColorStart', new InterleavedBufferAttribute(instanceColorBuffer, itemSize, 0)) // rgb(a)
+    this.setAttribute('instanceColorEnd', new InterleavedBufferAttribute(instanceColorBuffer, itemSize, 3)) // rgb(a)
```

Comment:
The last parameter should be `itemSize` instead of `3`. 

Source: https://threejs.org/docs/?q=interleaved#api/en/core/InterleavedBufferAttribute